data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,731 @@
+		goto tributeskip
+	label weak
+	`	"You apologize? After killing my men and defending territory you took with your own blood and sweat? That's..." The Capo looks like he's on the cusp of shouting at you or breaking into laughter. "Absolutely pathetic. If not for the fact that you killed my entire crew I'd have you erased and remove such a weak individual from the galaxy as a public service. Now, piss off. I've got nothing more to say to you."`
+	label defenderspeech
+	`	Meanwhile, the Corsair has some information for you.`
+	`	"Ye did well defending those poor folks, and they've already spread word of it. Protector of the downtrodden ain't a common reputation one finds in these parts, but it's... Well, a reputation nonetheless. The fact that ye thrashed two different gangs in the process mostlikely has somethin' to do with that, aye? If I had to hazard a guess, ye might find pirate folk treating you with a bit more respect after a display like that. Hells, I know some folk who'd be keen to call ye kin."`
+		goto tributeskip
+	label slaver
+	choice
+		`	(Address the Capo.) "I conquered Darkstone and sold the inhabitants into slavery for a tidy profit."`
+	action
+		set RookieSlaver
+	`	"So I heard, so I heard! It was my boys you were selling those dirt-grubbers off to. Mind you, I wasn't expecting to need to cut you in on this, but you did chase off the old owners so I suppose it's fair. One thing is sure: you've got a winner's instincts. I'm proud to sponsor you, and rest assured... people will know you're a cutthroat bastard, ready to sell any body that falls into your hands. Just like me." The Capo pauses to wipe a tear from his eye. "I swear, if I still had lips I'd be smiling."`
+	label outlawapproval
+	`	"Now, I'll do as I said and help make your name known to those who matter, and the lowlifes that don't. Pirates will respect you for the deeds you've done."`
+		goto tributeskip
+	#label ring
+	#`	The Capo extends his palm, a shining silver ring glistening in the center. Unadorned by any manner of stone, it's a simple, subdued piece of jewelry. Upon the front a crest has been stamped. "A token of my approval. Consider it sponsorship, and my trust that you'll do what need be done. Take it to Gienah. People will know what it means, and you'll find lucrative work that lines up nicely with the skillset you've demonstrated."`
+	#`	The cyborg offers a genial smile - or at least as much as one can without lips. It's uncanny how expressive articulated artificial jaws can be these days.`
+	label tributeskip
+	branch boardingskip
+		not "Pirate Intro: Boarding: done"
+	choice	
+		`	(Slap the top of the Corsair's trunk.) "I've brought the antique lockbox and defeated the other pirates who came for it."`
```

Comment:
The player's acts of violence are part of the story. Just because other campaigns have Captain Last be a soulless chatbot doesn't mean the Pirate Campaign has to.